doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -5,7 +5,8 @@
   instead of `pimcore_core`. If you want to handle image optimize messages, then it is required to add specific option `pimcore_image_optimize` to the command `bin/console messenger:consume pimcore_core pimcore_maintenance pimcore_image_optimize`. Also run command `bin/console messenger:consume pimcore_core` before the upgrade, so that ImageOptimize messages on the queue gets consumed.
 - **Important**: [Object bricks] A call to the object brick´s getter method no longer returns object bricks marked for deletion. 
   To restore the original behavior pass "true" to the getter method´s `$includeDeletedBricks` argument. 
-   
+- [Image Optimizer] Image Optimizer services (e.g. PngCrushOptimizer, JpegoptimOptimizer etc.) are deprecated and will be
```

Comment:
We should also mention that the existing optimizers were disabled by default and we should also describe here how to re-enable them again if necessary.